<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OngletPoints</class>
 <widget class="QMainWindow" name="OngletPoints">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>924</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Points</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/Points.png</normaloff>img/Points.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget {
    background-color:rgb(255, 255, 255);
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Recherche par nom de point :</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLineEdit" name="inputRecherchePoint">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>258</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QTableWidget" name="tableWidgetPoints">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>nom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>E [m]</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>N [m]</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>H [m]</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>th√®me MO</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>nature MO</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>commentaire</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNouveau"/>
   <addaction name="actionOuvrir"/>
   <addaction name="actionEnregistrer"/>
   <addaction name="actionEnregistrer_sous"/>
   <addaction name="separator"/>
   <addaction name="actionSupprimer_les_doublons"/>
  </widget>
  <action name="actionNouveau">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>img/newFile.png</normaloff>img/newFile.png</iconset>
   </property>
   <property name="text">
    <string>Nouveau...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="icon">
    <iconset>
     <normaloff>img/open2.png</normaloff>img/open2.png</iconset>
   </property>
   <property name="text">
    <string>Ouvrir...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionEnregistrer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>img/save.png</normaloff>img/save.png</iconset>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionEnregistrer_sous">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>img/saveAs.png</normaloff>img/saveAs.png</iconset>
   </property>
   <property name="text">
    <string>Enregistrer sous...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSupprimer_les_doublons">
   <property name="icon">
    <iconset>
     <normaloff>img/suppDoublons.png</normaloff>img/suppDoublons.png</iconset>
   </property>
   <property name="text">
    <string>Supprimer les doublons</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
